src/runtime/auto-prompt-manager.ts

```diff
@@ -214,6 +223,50 @@ export class AutoPromptManager {
       return;
     }
     // Set experiment flags here.
+    const articleExpFlags =
+      this.entitlementsManager_.parseArticleExperimentConfigFlags(article);
+    this.onsitePreviewEnabled_ = articleExpFlags.includes(
+      ArticleExperimentFlags.ONSITE_PREVIEW_ENABLED
+    );
+  }
+
+  /**
+   * Displays the appropriate auto prompt for onsite preview.
+   */
+  private async showPreviewAutoPrompt_(
+    article: Article,
+    params: ShowAutoPromptParams
+  ): Promise<void> {
+    const actions = article.audienceActions?.actions;
+    if (!actions || actions.length === 0) {
+      return;
+    }
+    // Article response is honored over code snippet in case of conflict, such
+    // as when publisher changes revenue model but does not update snippet.
+    this.autoPromptType_ = this.getAutoPromptType_(
```

Comment:
I'm not sure if this part is necessary. If it was added for the closabilty fallback logic, see my comment below.

Reply:
Both AudienceActionLocalFlow & AudienceActionIframeFlow are using autoPromptType_ & isClosable_ values, and the two values are used in other places as well. Not gonna change them all in this pr, we can probably clean them up later.